install/kubernetes/cilium/templates/hubble-ui/_nginx.tpl

```diff
@@ -1,7 +1,9 @@
 {{- define "hubble-ui.nginx.conf" }}
 server {
     listen       8081;
+{{- if .Values.hubble.ui.frontend.server.ipv6.enabled }}
```

Comment:
just curious if there is any issue having dual stack enabled by default 🤔

Reply:
Yes, this has been a problem several times already (https://github.com/cilium/hubble-ui/issues/278)

Reply:
Do you think it's better to make it disabled by default? That would be a change that potentially can break some current deployments I think.